### PR TITLE
feat: connect db to project

### PR DESCRIPTION
**Pull request by LBokan**
### Description

Connected DB to the project. Installed Liquibase and set it up

### Type of change

- [x] Feature (introduces new functionality)
- [ ] Bugfix
- [ ] Configuration
- [ ] Refactoring (doesn't change functionality, only the code)
- [ ] Style changes (doesn't affect functionality, only the look of the components)

### PR Checklist

- [x] Synced up with latest from the main branch/Code is linted
- [x] Description of the changes has been added (Screenshots/Video)

> If not explain here

- [x] Commit messages follow [conventional commits guideline](https://www.conventionalcommits.org/en/v1.0.0-beta.4/#summary)
- [x] Console has been checked for warnings

#### Screenshots/Video before change

Add screenshots of the changed components here.

#### Screenshots/Video after change

Add screenshots with the changes applied here.

### CODE REVIEW

File: backend/pom.xml [hunk 1]
```diff
@@ -29,6 +29,10 @@
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-web</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>org.liquibase</groupId>
+			<artifactId>liquibase-core</artifactId>
+		</dependency>
```
Comment (rmaskalenka) [inline backend/pom.xml:35]: By adding liquibase in this way you are using it as a part of your code. It is not a best practice to use liquibase programmatically or make your actual service aware of utility tools such as liquibase.

Please don't use liquibase as source dependncy. Mavan should use liquibase as a plugin.

Example can be found here: https://rcarrillo.dev/setup-liquibase-on-maven/

File: backend/src/main/resources/db/changelog/0.0.1/changelog-1.sql
Comment (rmaskalenka) [inline backend/src/main/resources/db/changelog/0.0.1/changelog-1.sql:1]: By placing liquibase into `resources` folder you will save this file in your final compiled JAR as a part of your service. This is usually an unwanted approach as liquibase should be considered as independent tool and not part of the real service.

**Review by rmaskalenka — CHANGES_REQUESTED** [review]

File: backend/pom.xml (hunk 1 above)
Comment (LBokan) [reply backend/pom.xml]: Done

File: backend/src/main/resources/db/changelog/0.0.1/changelog-1.sql
Comment (LBokan) [reply backend/src/main/resources/db/changelog/0.0.1/changelog-1.sql:1]: Done. Removed changelogs from resources to the project root

**Review by rmaskalenka — APPROVED** [review]

**Pull request merged**